src/HeaderMenu/i18n/de.js

```diff
@@ -1,5 +1,6 @@
 export default {
   HeaderMenu: {
+  	search: 'Suche',
```

Comment:
Followed the same indentation from the other translation files. Could you tell me what needs to be changed?